src/common/utils.ts

```diff
@@ -22,7 +22,11 @@ $.prefix = 'set -euo pipefail;' // https://github.com/google/zx/blob/main/index.
 export const rootDir = process.cwd() === '/home/app/stack/env' ? '/home/app/stack' : process.cwd()
 export const parser = yargs(process.argv.slice(3))
 export const getFilename = (path: string): string => fileURLToPath(path).split('/').pop()?.split('.')[0] as string
-
+// A kubernetes secret that contains generated passwords during at bootstrap stage. Used only during helm chart deployment of otomi.
+export const otomiPasswordsSecretName = 'otomi-generated-passwords'
+export const otomiPasswordsNamespace = 'default'
+export const otomiStatusCmName = 'otomi-status'
+export const otomiStatusNamespace = 'default'
```

Comment:
Can we rename these also? The prefix can be dropped as it serves no purpose imo.

At the same time, should we not use CAPS for global constants that get imported?

And please don't use abbreviations as not everybody understands what `cm` stands for.

I suggest the following names:
```ts
DEPLOYMENT_PASSWORDS_SECRET_NAME
DEPLOYMENT_PASSWORDS_SECRET_NAMESPACE
DEPLOYMENT_STATUS_CONFIGMAP_NAME
DEPLOYMENT_STATUS_CONFIGMAP_NAMESPACE
```

Reply:
If you can set linter to enforce this policy then sure, if not then let's not make this gentlemen agreements and focus on things that really matters.

Reply:
linters are not always suitable for style guide enforcement...you should know

Reply:
this is not bash, we have modules and there are no globals. Vscode immediately guide about type of this object. Not an issue imo.